cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -129,6 +129,9 @@
     # LD_PRELOAD to be set to a library which is in a directory configured by
     # ld.so.conf, but access to those locations is mediated by this profile
     # (which requires rules for specific locations).
+    # TODO: use GenerateAAREExclusionPatterns for this, though the first
```

Comment:
Yes but it would still be nice to have some way to use the Go code which does this to generate it instead like perhaps as a `go generate` thing, hence the TODO